src/test/java/seedu/address/logic/commands/patient/ClearPatientCommandTest.java

```diff
@@ -23,13 +26,21 @@ public void execute_emptyPatientRecords_success() {
 
     @Test
     public void execute_nonEmptyPatientRecords_success() {
```

Comment:
should edit the method name to indicate that this method is for empty appointmentschedule, non empty patient records